Chat/ClientApp/src/components/Room.tsx

```diff
@@ -19,7 +19,7 @@ export default (props: RoomProps): JSX.Element => {
     <Stack className={chatScreenContainerStyle}>
       <ChatHeader leaveChatHandler={() => { }} selectedPane={SidePanelTypes.None} setSelectedPane={() => { }} />
       <Stack className={chatScreenBottomContainerStyle} horizontal={true}>
-        <LeftPane />
+        <LeftPane onTabClicked = {() => {} }/>
```

Comment:
I will be getting rid of this file :)

Reply:
Yep this will be removed since your PR has been merged